expertise/create_dataset.py

```diff
@@ -36,12 +36,31 @@ def convert_to_list(self, config_invitations):
         assert isinstance(invitations, list), 'Input should be a str or a list'
         return invitations
 
+    def get_paper_notes(self, author_id, dataset_params):
+
+        use_bids_as_expertise = dataset_params.get('bid_as_expertise', False)
+
+        if use_bids_as_expertise:
+            bid_invitation = dataset_params['bid_invitation']
+            bids = openreview.tools.iterget_edges(self.openreview_client, invitation=bid_invitation, tail=author_id)
```

Comment:
Is the `author_id` always going to be in the `tail`?

Reply:
Usually for bids, head is the paper id and tail is the profile id.